### PR TITLE
21Clock

### PR DESCRIPTION
**Pull request by timohahaaa**
https://timohahaaa.github.io/layout_stop-watch/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -9,6 +9,10 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Stop watch</h1>
+    <div class="clock">
+      <div class="center-circle"></div>
+      <div class="hour-hand"></div>
+      <div class="minute-hand"></div>
```
Comment (polosanya) [inline src/index.html:15]:
```suggestion
    <div class="clock">
      <div class="clock__center-circle"></div>
      <div class="clock__hour-hand"></div>
      <div class="clock__minute-hand"></div>
```
image

File: src/styles/main.scss [hunk 2]
```diff
@@ -1 +1,61 @@
-// put code here
+.clock {
```
Comment (polosanya) [inline src/styles/main.scss:1]: image

File: src/styles/main.scss [hunk 3]
```diff
@@ -1 +1,61 @@
...
+  margin: auto;
```
Comment (polosanya) [inline src/styles/main.scss:2]: image

**Review by polosanya — CHANGES_REQUESTED** [review]
Nice! Please pay attention [to the checklist ](https://github.com/mate-academy/layout_stop-watch/blob/master/checklist.md)

**Review by maxim2310 — APPROVED** [review]
Good Work!!!

**Pull request closed without merging**